### DISCUSSION TITLE
Having Troubles with big vector array

### DISCUSSION THREAD

**Discussion by VimForLaNie** [cpeditor/cpeditor#1010] [Q&A] [answered]
I was writing Dijkstra's algorithm in C++ as a practice but encountered the problem when I try to allocate 
`vector<pair<long long,long long>> graph[100005];`
It compiles well but when I try running it, it takes too long and exits with errorcode " 62097 " 
but when I submit the code. It was fine. No errors at all.
When I lower the size of the graph array, it runs fine and with no issues.

** compile command (don't know if it even matters, I try changing it, same problem)
`g++ -DEVAL -std=gnu++11 -O2 -pipe -static -s`

**Comment by ouuan** [comment, marked as answer]
First of all, it's not related to CP Editor. You should get the same result if you compile the code in the terminal.

And we need the whole code to identify the problem.

**Reply by VimForLaNie** [reply]
I try running the code on another machine with different compiler and it works, so I think it's my compiler that caused the problem.

Sorry to bother you!
